backend/runners/load_representative_points.py

```diff
@@ -44,8 +52,19 @@ def _main(kwargs):
     with open(kwargs['filepath'], 'r') as f:
         json_features = geojson.load(f)['features']
 
-    for feature in json_features:
-        feature['properties']['zip_code'] = FAKE_ZIP_CODE
+    if FAKE_ZIP_CODE:
+        for ft in json_features:
+            ft['properties']['zip_code'] = FAKE_ZIP_CODE
+
+    # If a fake state is used, fake all data.
+    if kwargs['fake_state']:
+        for ft in json_features:
+            ft['properties']['zip_code'] = FAKE_ZIP_CODE or '00000'
+            ft['properties']['state'] = kwargs['fake_state']
+            ft['properties']['county'] = ft['properties']['state'] + '_' + FAKE_COUNTY
```

Comment:
So service area ids will be `ca_ca_alameda_0000`?

Reply:
Note that we already have duplicate counties (in different states), so I don't think we have to worry about including the state in the county field.

Reply:
We could also have `FAKE_COUNTY` default to `fake_state`.